.github/workflows/deploy-to-azure-app-service.yml

```diff
@@ -90,6 +80,10 @@ jobs:
         timeout-minutes:
         - ${{ inputs.timeout-minutes }}
     timeout-minutes: ${{ matrix.timeout-minutes }}
+    env:
+      publish-command-base: >-
+        dotnet publish (Get-ChildItem ${{ inputs.web-project-path }}).FullName --configuration Release
+        --output '${{ inputs.build-directory }}/Published' --verbosity ${{ inputs.build-verbosity }}
```

Comment:
This is an interesting approach, but my issue is that we're reinventing a lot of wheels from `build-dotnet`. Instead, I'd keep using it and publish with `no-build` (the two together should take basically the same amount of time then just publish with build). Even if needs adding an option to `build-dotnet` to build a csproj instead of a solution.